dev/Repeater/ViewManager.cpp

```diff
@@ -731,6 +752,7 @@ winrt::UIElement ViewManager::GetElementFromElementFactory(int index)
             }();
 
             elementAsFE.DataContext(elementDataContext);
+            virtInfo->MustClearDataContext(true);
```

Comment:
Please add a test. Thanks!

Reply:
Yes of course, just wasn't sure how until your latest comment on #2384 .

Reply:
Added a test now, unfortunately it has quite a large setup to inject the correct elements and allow the recycling to take place.